runelite-client/src/main/java/net/runelite/client/plugins/zalcano/ZalcanoPlugin.java

```diff
@@ -132,6 +161,11 @@ public void onNpcDespawned(NpcDespawned event)
 		if (npc.getId() == ZALCANO_WEAKENED)
 		{
 			client.clearHintArrow();
+
+			if (npc.isDead())
```

Comment:
If you were to leave in the middle of a round this would not reset, is there a better way to do that?